spec/factorial_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rspec'
+require 'rspec/mocks/standalone'
+require_relative '../solver'
+
+RSpec.describe Solver do
```

Comment:
- [ ] Since you already have ```rspec-core``` in your gem file, you do not need to call ```RSpec.describe```. You can just call the ```describe``` block directly